.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
```

Comment:
I don't quite understand what the purpose of this file is, seeing that all our tests are going to be run through django. I do like having flake8 check our code style though.

Reply:
If we're doing flake8, may as well run black as well.

Reply:
@tyxchen the purpose is this was the default Python CI GitHub Actions script. I've removed the pytest part, but kept the code style checks.